Providers/Modules/NPM/Plugin/plugin/npmd_config_lib.rb

```diff
@@ -484,8 +485,8 @@ def self.parse(string)
                 _h[KeyAgents]   = getAgentHashFromJson(_config.elements[AgentInfoTag].text())
                 _h[KeyRules]    = getRuleHashFromJson(_config.elements[RuleInfoTag].text()) unless _config.elements[RuleInfoTag].nil?
                 _h[KeyEpm]      = getEpmHashFromJson(_config.elements[EpmInfoTag].text()) unless _config.elements[EpmInfoTag].nil?
+                _h[KeyCM]       = JSON.parse(_config.elements[EpmInfoTag].text())[KeyCM] unless _config.elements[EpmInfoTag].nil?
```

Comment:
I would suggest to have a method for this too, and handle JSON parse exceptions there to avoid any failures.

Reply:
updated